specs-go/v1/descriptor.go

```diff
@@ -44,6 +44,9 @@ type Descriptor struct {
 	//
 	// This should only be used when referring to a manifest.
 	Platform *Platform `json:"platform,omitempty"`
+
+	// ArtifactType is the IANA media type of this artifact.
+	ArtifactType string `json:"artifactType,omitempty"`
```

Comment:
?

Reply:
This is used in the Referrers listing.

Reply:
if this refers descriptor example is correct, the definition could be improved by saying
```suggestion
        // a refers descriptor use of artifactType SHOULD be to specify the media type of the artifact being referred to
	ArtifactType string `json:"artifactType,omitempty"`
```

Reply:
needs some explanation regarding the parts of the spec that are descriptors and what the artifact type field currently means in those contexts.. will try to draft something probably needs more examples